web/app/src/App.css

```diff
@@ -1,5 +1,10 @@
 .content {
   display: flex;
-  justify-content: center;
+  justify-content: space-between;
```

Comment:
Before | After
----------|--------------
<img width="652" alt="image" src="https://user-images.githubusercontent.com/508822/188328507-68d68961-d5cd-40e1-9521-22c0332894a1.png"> | <img width="659" alt="image" src="https://user-images.githubusercontent.com/508822/188328495-3370898d-d1a5-4816-885b-f498851f12cf.png">

In the future, we might be able to specify "[safe](https://www.w3.org/TR/css-align-3/#overflow-values) center", but currently, few browsers support the feature.
cf. https://caniuse.com/mdn-css_properties_justify-content_flex_context_safe_unsafe